tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -566,16 +568,17 @@ def test_unsupported_ipu_choice(monkeypatch):
 
 
 def mock_hpu_available(monkeypatch, value=True):
-    monkeypatch.setattr(lightning.pytorch.accelerators.hpu, "_HPU_AVAILABLE", value)
-    monkeypatch.setattr(lightning.pytorch.accelerators.hpu.HPUAccelerator, "is_available", lambda: value)
-    monkeypatch.setattr(lightning.pytorch.strategies.hpu_parallel, "_HPU_AVAILABLE", value)
-    monkeypatch.setattr(lightning.pytorch.strategies.single_hpu, "_HPU_AVAILABLE", value)
-    monkeypatch.setattr(lightning.pytorch.plugins.precision.hpu, "_HPU_AVAILABLE", value)
+    import lightning_habana
```

Comment:
This should return if it's not available

Reply:
```suggestion
    try:
        import lightning_habana
    except ModuleNotFoundError:
        return
```